boards/lolin_s3_pro.json

```diff
@@ -0,0 +1,50 @@
+{
+  "build": {
+    "arduino": {
+      "ldscript": "esp32s3_out.ld",
+      "partitions": "default_16MB.csv",
+      "memory_type": "qio_opi"
+    },
+    "core": "esp32",
+    "extra_flags": [
+      "-DARDUINO_LOLIN_S3_PRO",
+      "-DBOARD_HAS_PSRAM",
+      "-DARDUINO_USB_MODE=1",
+      "-DARDUINO_USB_CDC_ON_BOOT=1",
```

Comment:
Are you sure the `-DARDUINO_USB_CDC_ON_BOOT=1` macro is necessary? In the boards.txt it's [disabled](https://github.com/espressif/arduino-esp32/blob/master/boards.txt#L8140) by default.